Packs/Reco/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Reco
+- Updated the Docker image to: *demisto/python3:3.10.10.51930*.
+
+- Update version to 1.0.4
+- Add reco-get-sensitive-assets-by-id command
```

Comment:
```suggestion
##### Reco
- Updated the Docker image to: *demisto/python3:3.10.11.55362*.
- Added the ***reco-get-sensitive-assets-by-id*** command.
```

Reply:
note that the docker image should be updated to *demisto/python3:3.10.11.55362* in the yml file, line 68 as well.